modules/chess/src/main/scala/Clock.scala

```diff
@@ -254,10 +255,9 @@ object Clock {
     def limitString: String =
       limitSeconds match {
         case l if l % 60 == 0 => (l / 60).toString
-        case 15 => "¼"
         case 30 => "½"
-        case 45 => "¾"
```

Comment:
I can see, why you would want to get rid of these times for byo, but they might still be useful for inc. And then there is this whole thing of backward compatibility.

Reply:
Ah, right, increment... OK, yes between that and backward compatibility I'll revert the 1/4 and 3/4 change.